samples/features/security/ledger/azure-sql-database/setup/CreateDatabaseSchema.sql

```diff
@@ -0,0 +1,194 @@
+SET ANSI_NULLS ON
+GO
+
+SET QUOTED_IDENTIFIER ON
+GO
+
+CREATE TABLE [dbo].[Employees](
+	[EmployeeID] [int] IDENTITY(1,1) NOT NULL,
+	[SSN] [char](11) NOT NULL,
+	[FirstName] [nvarchar](50) NOT NULL,
+	[LastName] [nvarchar](50) NOT NULL,
+	[Salary] [money] NOT NULL
+	)
+WITH 
+(
+  SYSTEM_VERSIONING = ON,
+  LEDGER = ON
+); 
+GO
+
+
+CREATE TABLE [dbo].[AuditEvents](
+	[Timestamp] [Datetime2] NOT NULL DEFAULT (GETDATE()),
+	[UserName] [nvarchar](255) NOT NULL,
+	[Query] [nvarchar](4000) NOT NULL
+	)
+WITH (LEDGER = ON (APPEND_ONLY = ON));
+GO
+
+CREATE TABLE [dbo].[LedgerVerifications](
+	[Timestamp] [Datetime2] NOT NULL DEFAULT (GETDATE()),
+	[DigestLocations] [nvarchar](max) NOT NULL,
+	[Result] [nvarchar](255) NOT NULL,
+)
+WITH (LEDGER = ON (APPEND_ONLY = ON));
+GO
+
+CREATE PROCEDURE [dbo].[GetEmployeeLedgerEntries]
+AS 
+SET NOCOUNT ON 
+BEGIN
+	SELECT
+	t.[commit_time] AS [CommitTime] 
+	, t.[principal_name] AS [UserName]
+	, l.EmployeeId
+	, l.[SSN]
+	, l.[FirstName]
+	, l.[LastName]
+	, l.[Salary]
+	, l.[ledger_operation_type_desc] AS Operation
+	FROM [dbo].[Employees_Ledger] l
+	JOIN sys.database_ledger_transactions t
+	ON t.transaction_id = l.ledger_transaction_id
+	WHERE t.[commit_time] > DATEADD(MINUTE, -10, GETDATE())
+	ORDER BY t.commit_time DESC
+END;
+GO
+
+CREATE PROCEDURE [dbo].[VerifyLedger]
+AS 
+BEGIN
+	SET NOCOUNT ON 
+	DECLARE @digest_locations NVARCHAR(MAX) = (SELECT * FROM sys.database_ledger_digest_locations FOR JSON AUTO, INCLUDE_NULL_VALUES);
+	BEGIN TRY
+       EXEC sys.sp_verify_database_ledger_from_digest_storage @digest_locations;
+       INSERT INTO [dbo].[LedgerVerifications] ([DigestLocations], [Result]) VALUES (@digest_locations, N'Success')
+	END TRY
+	BEGIN CATCH
+       INSERT INTO [dbo].[LedgerVerifications] ([DigestLocations], [Result]) VALUES (@digest_locations, N'Failure')
+	END CATCH
+END;
+GO
+
+CREATE PROCEDURE [dbo].[GetLedgerVerifications]
+AS 
+SET NOCOUNT ON 
+BEGIN
+	SELECT * 
+	FROM [dbo].[LedgerVerifications]
+	WHERE [Timestamp] > DATEADD(MINUTE, -60, GETDATE())
+	ORDER BY [Timestamp] DESC
+END;
+GO
+
+CREATE PROCEDURE [dbo].[GetAuditEvents]
+AS 
+SET NOCOUNT ON 
+BEGIN
+	SELECT * 
+	FROM [dbo].[AuditEvents]
+	WHERE [Timestamp] > DATEADD(MINUTE, -10, GETDATE())
+	ORDER BY [Timestamp] DESC
+END;
+GO
+
+CREATE TABLE [AspNetRoles] (
```

Comment:
Consider separating the tables for AspNet app to a separate file.

Reply:
All the tables are used by the same app.
I don't see any reason or advantage to have a schema deploy in different files.
This only makes it more complex.